.gitignore

```diff
@@ -31,3 +31,6 @@
 
 # electron build
 dist
+
+# design system ts output
+/src/js/components/**/*.js
```

Comment:
For some reason, I my git is recognizing unstaged changes in `/src/js/components/Button/Button.js` and `/src/js/components/Button/Button.stories.js`. The diff is just different slightly differenent transpiled TS=>JS.

Reply:
What's your version of git? Recursive glob support was only added in 1.8.2.

Reply:
2.25.1. Can look into it and add my repro steps.

Reply:
Seems fine now. But `storybook-static/` code is being created. Do we want all these files?

Reply:
Ignored